### PR TITLE
 #451 disabled sources tab content on manage_stock update

### PR DESCRIPTION
**Pull request by AleksLi**
### Description

Assign Sources tab on product page in admin should be disabled if function "Manage Stock" in Advanced Inventory set to "No" for this product
### Fixed Issues (if relevant)

1. https://github.com/magento-engcom/msi/issues/451: Assign Sources tab should be disabled if function "Manage Stock" set "No" for product

### CODE REVIEW

**Comment by AleksLi** [comment]
I haven't fixed 2 bugs yet:
1. On page loading sources tab doesn't disable if `manage_stock` field is 0.
2. Even if everything is disabled but button remove source is still enabled and operate as usual.

**Comment by AleksLi** [comment]
I've updated this branch. Everything looks fine, except [this delete action button](http://joxi.ru/MAj3plec4MdOpr).
Can someone take a look at this?

**Comment by AleksLi** [comment]
I resolved conflicts rebasing to the current `develop` branch

File: app/code/Magento/InventoryCatalog/Ui/DataProvider/Product/Form/Modifier/SourceItems.php [hunk 1]
```diff
@@ -136,7 +149,50 @@ public function modifyMeta(array $meta)
...
+
+    /**
+     * Get "isManageStock" from product, and fall back to global config in case of new product.
+     *
+     * @param ProductInterface $product
+     * @return bool
+     */
+    private function isMangeStock(ProductInterface $product): bool
+    {
+        $item = $product->getExtensionAttributes() ? $product->getExtensionAttributes()->getStockItem() : null;
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Ui/DataProvider/Product/Form/Modifier/SourceItems.php]: Pls, use `\Magento\InventoryConfigurationApi\Api\GetStockItemConfigurationInterface` for loading configuration
You can use default stock id (`\Magento\InventoryConfigurationApi\Api\GetStockItemConfigurationInterface`)

Comment (AleksLi) [reply app/code/Magento/InventoryCatalog/Ui/DataProvider/Product/Form/Modifier/SourceItems.php]: Yes, no problem, I'll do that

Comment (nmalevanec) [reply app/code/Magento/InventoryCatalog/Ui/DataProvider/Product/Form/Modifier/SourceItems.php]: Done.

**Review by larsroettig — APPROVED** [review]

**Pull request merged**